ci_framework/playbooks/01-bootstrap.yml

```diff
@@ -25,6 +25,13 @@
       ansible.builtin.import_role:
         name: repo_setup
 
+    - name: Ensure artifacts parameters dir exists
```

Comment:
please move that to the ci_setup role

Reply:
Done, but I needed to run it before the install yamls role to have the directory present